### PR TITLE
Metadata: JPEXS.FFDec version 18.3.4

### PR DESCRIPTION
**Pull request by jindrapetrik**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95516)

### CODE REVIEW

**Comment by jindrapetrik** [comment]
@microsoft-github-policy-service agree

**Comment by jindrapetrik** [comment]
I am sorry for bothering you, but how long does it usually take to review a PR? A month or more?
I already released a new version in the meantime and would like later make a new PR for it.
Is the wait so long only for first time submission? Or every new version I release/PR will need to wait so long before accepting?
(Sorry for asking stupid questions, I am just impatient)

Thank you for review in advance!

File: manifests/j/JPEXS/FFDec/18.3.4/JPEXS.FFDec.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+FileExtensions:
+- abc
+- gfx
+- iggy
+- swc
+- swf
```
Comment (stephengillie) [inline manifests/j/JPEXS/FFDec/18.3.4/JPEXS.FFDec.installer.yaml:19]:
```suggestion
- swf
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**